dcgm/README.md

```diff
@@ -0,0 +1,260 @@
+
+```
+instances:
+
+    ## @param openmetrics_endpoint - string - required
+    ## The URL exposing metrics in the OpenMetrics format.
+    ##
+    ## Set this to <listenAddress>/<handlerPath> as configured in your DCGM Server
+    #
+    - openmetrics_endpoint: http://localhost:9400/metrics
+```
+
+<!-- xxx tab xxx -->
+<!-- xxx tab "Docker" xxx -->
+
+#### Docker
+
+1. To configure this check for an Agent running on a container:
+
+##### Metric collection
+
+Set [Autodiscovery Integrations Templates][5] as Docker labels on your application container:
+
+```yaml
+LABEL "com.datadoghq.ad.check_names"='["dcgm"]'
+LABEL "com.datadoghq.ad.init_configs"='[{}]'
+LABEL "com.datadoghq.ad.instances"='[{"openmetrics_endpoint": "http://%%host%%:9400/metrics"}]'
+```
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Kubernetes" xxx -->
+
+#### Kubernetes
+
+1. To configure this check for an Agent running on Kubernetes:
+
+##### Metric collection
+
+Set [Autodiscovery Integrations Templates][12] as pod annotations on your application container. Aside from this, templates can also be configured with [a file, a configmap, or a key-value store][11].
+
+**Annotations v1** (for Datadog Agent < v7.36)
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/dcgm.check_names: '["dcgm"]'
+    ad.datadoghq.com/dcgm.init_configs: '[{}]'
+    ad.datadoghq.com/dcgm.instances: |
+      [
+        {
+          "openmetrics_endpoint": "http://%%host%%:9400/metrics"
+        }
+      ]
+spec:
+  containers:
+    - name: dcgm
+```
+
+**Annotations v2** (for Datadog Agent v7.36+)
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/dcgm.checks: |
+      {
+        "dcgm": {
+          "init_config": {},
+          "instances": [
+            {
+              "openmetrics_endpoint": "http://%%host%%:9400/metrics"
+            }
+          ]
+        }
+      }
+spec:
+  containers:
+    - name: dcgm
+```
+
+
+<!-- xxz tabs xxx -->
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `dcgm` under the Checks section.
+
+
+### Adjusting Monitors
+
+The monitors that come with this integration out of the box have some common-sense default values for their alert thresholds. For instance, the GPU temperature was determined based on [acceptable range for industrial devices][13].
+However we recommend you check to make sure these values fit your particular needs.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this integration.
+
+### Events
+
+The DCGM integration does not include any events.
+
+### Service Checks
+
+The dcgm integration does not include any service checks.
+
+See [service_checks.json][7] for a list of service checks provided by this integration.
```

Comment:
Remove this line if service checks are not included

Reply:
Hmm, but we do provide service checks. If you have a look at the `dcgm/assets/service_checks.json` you'll see one listed.

Reply:
In line 210 we mention that the integration does not include service checks

`The dcgm integration does not include any service checks.`

Reply:
Oh wow, nice catch!!! Removed that line